.gitignore

```diff
@@ -0,0 +1,90 @@
+# Xcode
```

Comment:
Hell yeah, ignore all day.  I still need to make one of these for my project as well